Lib/collections/__init__.py

```diff
@@ -1527,6 +1527,29 @@ def isupper(self):
 
     def join(self, seq):
         return self.data.join(seq)
```

Comment:
This function is overwritten now. It's a bit difficult to spot because of the long comment.

Reply:
It's not overwritten, it's overloaded.

Reply:
Python doesn't have overloading!